src/Appwrite/SDK/Method.php

```diff
@@ -0,0 +1,256 @@
+        }
+
+        $descPath = \realpath(__DIR__ . '/../../../' . $desc);
+
+        if (!\file_exists($descPath)) {
+            self::$exceptions[] = "Error with {$this->getDebugName()} method: Description file not found at {$desc}";
+            return;
+        }
+    }
+
+    private function validateResponseModel(string|array $responseModel): void
+    {
+        $response = new Response(new HttpResponse());
+
+        if (\is_array($responseModel)) {
+            foreach ($responseModel as $model) {
+                try {
+                    $response->getModel($model);
+                } catch (\Exception $e) {
+                    self::$exceptions[] = "Error with {$this->getDebugName()} method: Invalid response model, make sure the model has been defined in Response.php";
+                }
+            }
+
+            return;
+        }
+
+        try {
+            $response->getModel($responseModel);
+        } catch (\Exception $e) {
+            self::$exceptions[] = "Error with {$this->getDebugName()} method: Invalid response model, make sure the model has been defined in Response.php";
+        }
+    }
+
+    public function getNamespace(): string
+    {
+        return $this->namespace;
+    }
+
+    public function getMethodName(): string
+    {
+        return $this->name;
+    }
+
+    public function getDescription(): string
+    {
+        return $this->description;
+    }
+
+    public function getAuth(): array
+    {
+        return $this->auth;
+    }
+
+    public function getResponses(): array
+    {
+        return $this->responses;
+    }
+
+    public function getResponseType(): ResponseType
+    {
+        return $this->responseType;
+    }
+
+    public function getMethodType(): ?MethodType
+    {
+        return $this->methodType;
+    }
+
+    public function getOfflineKey(): ?string
+    {
+        return $this->offlineKey;
+    }
+
+    public function getOfflineModel(): ?string
+    {
+        return $this->offlineModel;
+    }
+
+    public function getOfflineResponseKey(): ?string
+    {
+        return $this->offlineResponseKey;
+    }
+
+    public function isDeprecated(): bool
+    {
+        return $this->deprecated;
+    }
+
+    public function isHidden(): bool|array
+    {
+        return $this->hide ?? false;
+    }
+
+    public function isPackaging(): bool
+    {
+        return $this->packaging;
+    }
+
+    public function getRequestType(): string
+    {
+        return $this->requestType;
+    }
+
+    public function getParameters(): array
+    {
+        return $this->parameters;
+    }
+
+    public function getMultiplex(): array
+    {
+        return $this->multiplex;
+    }
+
+    // Throw any errors that were found during initialization
+    static function finaliseInitialization(): void
+    {
+        if (!empty(self::$exceptions)) {
+            throw new \Exception('Errors found during SDK initialization:' . PHP_EOL . implode(PHP_EOL, self::$exceptions));
+        }
+    }
```

Comment:
This is hacky, let's find a better way to do it. Should be an init check after all controllers are loaded